### PR TITLE
Updated Schedule in index.html 

### PR DESCRIPTION
**Pull request by Jeewal**
Updated Schedule as In the Linked PDF in index.html of Issue https://github.com/fossasia/2012.fossasia.org/issues/165

### CODE REVIEW

**Comment by Jeewal** [comment]
@YashKumarVerma

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]
Preview Link missing

**Comment by Jeewal** [comment]
https://jeewal.github.io/2012.fossasia.org/ @YashKumarVerma

**Review by YashKumarVerma — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -553,12 +553,12 @@ <h1>OpenDesignWeeks, April, 2-10, Ho Chi Minh City
...
                                     <span class="time">Sunday, April 8</span>
                                     <span class="title">
-                                        Workshops with students and trip to Mekong Delta and Can Tho City
+                                        Mekong Excursion and arrival in Cantho City
                                     </span>
                                 </div>
 
                                 <div class="schedule-text">
-                                    <p>Workshop sessions / Visit to designer labs and companies in HCMC</p>
+                                    <p> 7:00AM- 7:00PMExcursion and gathering material "Between Folk Art and Industrial production" in Mekong and arrival in Cantho </p>
```
Comment (simsausaurabh) [inline index.html:560]: Keep space between time and text.

File: index.html [hunk 2]
```diff
@@ -599,12 +602,18 @@ <h1>Open Design Weeks, April 9-15, 2012, Can Tho City
...
                                     <span class="title">
+
+                                        Workshops : Font Design
                                         Workshops and discussions
+
                                     </span>
                                 </div>
 
                                 <div class="schedule-text">
+
```
Comment (simsausaurabh) [inline index.html]: Keep space between time and text.

File: index.html [hunk 3]
```diff
@@ -599,12 +602,18 @@ <h1>Open Design Weeks, April 9-15, 2012, Can Tho City
...
+                                    <p>9:00AM-12:00PM and1:30PM-5:00PM Libre Graphics and Open Source Design Workshops and Sessions
```
Comment (simsausaurabh) [inline index.html:606]: Please follow up in whole file and fix it at all places.

**Pull request closed without merging**